test/js_test.go

```diff
@@ -970,6 +986,27 @@ func TestJetStreamImportDirectOnly(t *testing.T) {
 		}
 	}
 
+	// Do push based consumer using a regular NATS subscription on the import subject.
+	sub, err = nc.SubscribeSync("p.d3")
```

Comment:
Regular subscribe here is the same as this:
```
sub, err = js.SubscribeSync("ORDERS", nats.PushDirect("p.d3"))
```
so with this change we could remove `PushDirect` as well potentially.

Reply:
I think we should.